tests/lib/test_openstack_identity.py

```diff
@@ -172,6 +173,15 @@ def test_find_mandatory_project_raises_for_missing(self):
         self.instance.find_project = Mock(return_value=None)
         self.instance.find_mandatory_project(NonCallableMock(), NonCallableMock())
 
+    def test_list_projects(self):
+        """
+        Tests calling the API wrapper without a domain will throw
```

Comment:
The doc string is incorrect for this? As it doesn't throw (or we don't check it does)